lib/diagnostic.js

```diff
@@ -38,19 +50,33 @@ let owner = null;
 // Methods: vocalize
 
 // Replace null with your answer
-let pet = null;
+let pet = {
+  name: 'Hugo',
+  species: 'dog',
+  breed: 'terier',
+  vocalization: 1,
+  vocalize: function () {
+    return console.log(4*5);
+  }
+};
 
 // 4. Create a constructor function for pet objects, as described above. The
 // constructor function should be responsible for setting the values of `name`,
 // `species`, `breed`, and `vocalization`.
 
-const Pet = function () {
-  /* Your answer here */
+const Pet = function (name, species, breed, vocalization) {
+  this.name = name;
+  this.species = species;
+  this.breed = breed;
+  this.vocalization = vocalization;
 };
 
 // 5. Attach a `vocalize` method to the Pet's prototype.
 
 /* your answer here */
+Pet.prototype.vocalize = function () {
+console.log("The pet " + this.name + "and of breed " + this.breed);
+} ;
 
```

Comment:
I think I can see what you were going for here. PLease review the solution branch. 

```js
const vocalize = function () {
  return 'bark';
};

Pet.prototype.vocalize = vocalize;
```